src/fragments/lib-v1/storage/js/remove.mdx

```diff
@@ -5,41 +5,36 @@ Delete stored data from the storage bucket.
 ## Public level remove
 
 ```typescript
-import { remove } from 'aws-amplify/storage'
-
-async function removeFile() {
-	try {
-		await remove({ key: filename });
-	} catch (error) {
-		console.log('Error ', error);
-	}
-};
+import { remove } from 'aws-amplify/storage';
+
+try {
+	await remove({ key: filename });
+} catch (error) {
+	console.log('Error ', error);
+}
+
 ```
 
 ## Protected level remove
 
 ```typescript
-import { remove } from 'aws-amplify/storage'
-
-async function removeFile() {
-	try {
-		await remove({ key: filename, options: { accessLevel: 'protected' } });
-	} catch (error) {
-		console.log('Error ', error);
-	}
-}; 
+import { remove } from 'aws-amplify/storage';
+
+try {
+	await remove({ key: filename, options: { accessLevel: 'protected' } });
```

Comment:
I haven't build this branch locally but the indentation here looks different. But I'm totally OK if they look the same in the doc site.